packages/terra-base/src/Base.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import './Base.scss';
+
+const propTypes = {
+  /**
+   * The component(s) that will be wrapped by `<Base />`.
+   */
+  children: PropTypes.node.isRequired,
+};
+
+class Base extends React.Component {
```

Comment:
Since this component doesn't have state, we should probably make this a functional component.